deploy/package.sh

```diff
@@ -8,6 +8,8 @@ PRJ_ROOT="$(cd `dirname "${BASH_SOURCE}"`/..; pwd)"
 ENV_CODE=${1:-${ENV_CODE}}
 PIPELINE_NAME=${2:-${PIPELINE_NAME}}
 
+AI_MODEL_INFRA_TYPE=${3:-${AI_MODEL_INFRA_TYPE:-"batch-account"}} # Currently supported values are aks and batch-account
```

Comment:
We don't want this flag in this PR. We have a separate PR that we address this flag. For merge, let's not include this  flag in this PR.

Reply:
Removed this flag.